lib/to_do_app_web/router.ex

```diff
@@ -20,6 +20,21 @@ defmodule ToDoAppWeb.Router do
     get "/", PageController, :home
   end
 
+  scope "/", ToDoAppWeb do
+    pipe_through :browser
+
+    resources "/tasks", TaskController do
```

Comment:
https://github.com/dwyl/phoenix-todo-list-tutorial/blob/main/lib/app_web/router.ex

refer this and rewrite the router paths.

![image](https://github.com/Sad-lie/to_do_app/assets/66151658/cbfdc8d8-76a2-4b95-bb8a-8fc4e8e837ae)

https://hexdocs.pm/phoenix/routing.html#resources

refer this . When you use resources . it would auto generate a few predictable paths for CRUD operations. If you want to select a few of them or exclude a few of them you can use 

```elixir
resources "/posts", PostController, only: [:index, :show]
```

OR 

```elixir
resources "/comments", CommentController, except: [:delete]
```

So rethink how you can restruct and write the router paths.